selfdrive/ui/sidebar.cc

```diff
@@ -30,18 +26,14 @@ static void ui_draw_sidebar_network_strength(UIState *s) {
       {cereal::ThermalData::NetworkStrength::MODERATE, 3},
       {cereal::ThermalData::NetworkStrength::GOOD, 4},
       {cereal::ThermalData::NetworkStrength::GREAT, 5}};
-  const int network_img_h = 27;
-  const int network_img_w = 176;
-  const int network_img_x = !s->scene.uilayout_sidebarcollapsed ? 58 : -(sbr_w);
-  const int network_img_y = 196;
```

Comment:
Let's leave these. I think it's more readable, and we do it all over the place in the UI code.